source/updateCheck.py

```diff
@@ -119,7 +119,8 @@ def checkForUpdate(auto: bool = False) -> Optional[Dict]:
 		"version": versionInfo.version,
 		"versionType": versionInfo.updateVersionType,
 		"osVersion": winVersionText,
-		"x64": os.environ.get("PROCESSOR_ARCHITEW6432") == "AMD64",
+		"x64": os.environ.get("PROCESSOR_ARCHITEW6432") in ("AMD64", "ARM64"),

```

Comment:
this change should be safe

Reply:
It looks like we should also include `IA64`. 
Maybe worth referencing this document as well: https://docs.microsoft.com/en-gb/windows/win32/winprog64/wow64-implementation-details

```suggestion
		"x64": os.environ.get("PROCESSOR_ARCHITEW6432") in ("AMD64", "ARM64", "IA64"),
```